src/helix-ui/styles/components/forms/FieldName.less

```diff
@@ -0,0 +1,8 @@
+.FieldName() {
+  color: @gray-800;
+  font-size: 0.875rem; // ~14px
+}
+
+.FieldName(disabled) {
```

Comment:
We take advantage of pattern matching to overload LESS mixins (multiple mixins with same name, but different arguments).